### ISSUE TITLE
Gray 5 color fails color contrast ratio test

### ISSUE THREAD

**Issue by vikas-parashar** [yeun/open-color#53]
In the instructions [Here](https://yeun.github.io/open-color/gray-color.html) gray 5 is recommended for text(with white background) which fails WCAG 2.0 color contrast test, hence making it inaccessible for anyone with low vision impairments and color deficiencies. 
![screen shot 2017-01-10 at 11 06 03 pm](https://cloud.githubusercontent.com/assets/5694967/21818101/8a28de4a-d78c-11e6-9ee4-2f231ba26c51.png)

**Comment by yeun** [comment]
Thanks for reporting. I will improve it in version 2.

**Comment by pspeter3** [comment]
Are there accessibility guidelines based on the delta between shades?
